pkg/api/dashboard.go

```diff
@@ -142,7 +143,7 @@ func (hs *HTTPServer) GetDashboard(c *contextmodel.ReqContext) response.Response
 		creator = hs.getUserLogin(c.Req.Context(), dash.CreatedBy)
 	}
 
-	annotationPermissions := &dtos.AnnotationPermission{}
+	annotationPermissions := &v0alpha1.AnnotationPermission{}
```

Comment:
random thought, should we standardize on renaming these imports to something more descriptive (in this case maybe `dashboards`) vs using the raw version we get by default?

Reply:
except that we have ~3 "dashboards" packages so have to keep aliasing them.

I am absolutely open to suggestions on what the imports should be!